sled-agent/src/sled_agent.rs

```diff
@@ -360,15 +360,21 @@ impl SledAgent {
 
```

Comment:
Do we still need this initial notification? Why just *only* send the latter notification?

(If there's a good reason, we can notify Nexus both times, but it seems simpler to only do it once?)

Reply:
I don't know any reason specifically why we'd _need_ the initial one (and in fact it causes a lot of log noise if the sled in question is the one that's going to be hosting nexus and thus it doesn't yet exists).